Week1/exercise/w1/index.js

```diff
@@ -1,30 +1,53 @@
 console.log('Hack your future, right?!');
 
-const header = document.querySelector('h1');
-console.log('header: ', header);
+const selectHeader = document.querySelector('h1');
+console.log('header: ', selectHeader);
 
 // EXERCISE:
 // select "change image" button (querySelector)
+const selectButtonChangeImage = document.getElementById('btn-changeImage');
+
+// define changeImage() function
+const changeImage = () => {
+  // in this function:
+  //    get image element
+  const selectImage = document.querySelector('img');
+  //    get imageInput element
+  const selectImageInput = document.getElementById('imageInput');
+  //    log input value
+  console.log(selectImageInput.value);
+  //    set image src to imageInput element value
+  selectImage.src = selectImageInput.value;
+}
 // add click event listener
 // execute changeImage() function on click event
-// define changeImage() function
-// in this function:
-//    get image element
-//    get imageInput element
-//    log input value
-//    set image src to imageInput element value
+selectButtonChangeImage.addEventListener('click', changeImage);
+
+
 
 // ====================================== //
 
 // BONUS:
 // select "add todo" button (querySelector)
+const getToDoButton = document.querySelector('#btn-addTodo');
```

Comment:
the 'get' part of the variable indicates it will DO something, like a function. But in fact it is just the button. So just todoButton would be a better name here